src/main/java/org/openstreetmap/atlas/utilities/runtime/RunScript.java

```diff
@@ -22,12 +22,12 @@
 
     public static void run(final String command)
     {
-        run(new String[] { command });
+        run(command.split("\\s+"));
```

Comment:
Just worth noting, the regex `\s+` will effectively match `[\r\n\t\f\v ]+`. The old `run` method split against `[ \t\n\r\f]` thru Java's `StringTokenizer` class. Considering `\v` is a vertical tab character, I doubt this will cause any breakages in client code (nobody uses vertical tabs).

Reply:
If you wanted to perfectly match what the old code did, this line should become:
```java
run(command.split("[ \t\n\r\f]"))
```

Reply:
Great!

Reply:
I've learned a lot about regex last few days :)